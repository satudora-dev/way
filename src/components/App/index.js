import React from 'react';

class App extends React.Component {

  render() {
    this.props.history.push('./login')
    return (
      <div className="App">
        <h2>Now Loading ...</h2>
      </div>
    );
  }
}

export default App;
