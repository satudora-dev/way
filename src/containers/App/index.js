import React from 'react';

class App extends React.Component {
  componentWillMount(){
    this.props.history.push('/login');
  }

  render() {
    return (
      <div className="App">
        <h2>Now Loading ...</h2>
      </div>
    );
  }
}

export default App;
