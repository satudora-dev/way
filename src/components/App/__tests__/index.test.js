import React from 'react';
import App from '../../App';
import renderer from 'react-test-renderer';

test('App snapshot', () => {
  const component = renderer.create(
    <App />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
