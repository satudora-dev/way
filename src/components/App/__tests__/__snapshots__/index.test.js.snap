// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot 1`] = `
<div
  className="App"
>
  <h2>
    Now Loading ...
  </h2>
</div>
`;
